build-scripts/windist.ps1

```diff
@@ -0,0 +1,13 @@
+rm -Recurse -Force bindist
+
+mkdir bindist
+cp Cataclysm-vcpkg-static-Release-x64.exe bindist/cataclysm-tiles.exe
+cp tools/format/JsonFormatter-vcpkg-static-Release-x64.exe bindist/json_formatter.exe
+
+mkdir bindist/lang
+cp -r lang/mo bindist/lang
+
+$extras = "data", "doc", "gfx", "LICENSE.txt", "LICENSE-OFL-Terminus-Font.txt", "README.md", "VERSION.txt"
+ForEach ($extra in $extras) {
+	cp -r $extra bindist
+}
```

Comment:
I would probably put the `Compress-Archive` step inside this script rather than in the workflow.

Reply:
I had that at first, until I read about the upload-artifacts double zipping issue, so I wanted to leave option value open. But now that I know about release.yml I will look there instead and it’ll probably compress in this script again.